### ISSUE TITLE
Performance seems poor

### ISSUE THREAD

**Issue by s1037989** [KeystoneIT/Ama#42]

**Comment by s1037989** [comment]
There's a very real possibility that the server is simply under provisioned.  There are several containers on a single VM.  Does the VM have adequate resources assigned to it?  It's a single CPU with 1GB RAM!  For the host _and_ its _7_ containers (db, db-trusty, irc, kit, kt-wordpress, mojo, mosshe).

This problem was reported well before the addition of the mosshe container, but that container which runs a myriad of shell process every 5 minutes, seems to have really brought everything to its knees.  That's kind of why I came to the conclusion that under provisioned resources may be the issue.

**Comment by jsiems** [comment]
fixed #61

**Comment by s1037989** [comment]
#61 doesn't seem to be related to this... is that why @meyerben reopened this?

**Comment by jsiems** [comment]
Yes, I originally removed it because I did not notice the performance being bad at the time.

Ben decided to reopen this after we ran  a stress test and added 70000 questions to the server to see how well it would perform.  The results?  Bad. 

It took almost 2 min to receive all information from the server...

To solve this, my ideas- 
1. routinely delete unpopular questions
2. make sql queries faster somehow
3. get bigger computer to run server on
